reactive/ovn_chassis_charm_handlers.py

```diff
@@ -73,6 +73,22 @@ def enable_openstack():
         charm_instance.assess_status()
 
 
+@reactive.when(OVN_CHASSIS_ENABLE_HANDLERS_FLAG)
+@reactive.when_none('charm.installed')
```

Comment:
On a fresh install, is there any risk of a race where openvswitch-switch package isn't installed yet at this point? ie. we render the default file, restart ovs (does it fail if doesn't exist?), then ovs package gets installed and possibly overwrites default file?

Reply:
Yes, that's an excellent observation.

Reply:
You know what, it looks like I have painted myself into a corner with this "separate reactive from charm code" purity approach. I would have to override the default reactive install handler from layer-openstack etc etc to continue this line. I'll revert back to your approach and add a extra condition to check in the install method. Sorry about the noise and thank you for spending time reviewing! :)